.github/workflows/rotate-emoji.yml

```diff
@@ -0,0 +1,53 @@
+name: Rotate emojiData
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: rotate-emoji-group
+
+jobs:
+  sync-emojis-from-github:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node.js v20
+        uses: actions/setup-node@v3
```

Comment:
Why not [`actions/setup-node@4`](https://github.com/actions/setup-node/releases)?